### PR TITLE
Table styles

### PR DESCRIPTION
**Pull request by ivanzusko**

### CODE REVIEW

File: edit.svg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20">
```
Comment (nik-kor) [inline edit.svg]: is it the same src/assets/img/edit.svg ?

File: src/modules/buttons/Checked.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import './Checked.css';
+
+const Checked = () => {
```
Comment (nik-kor) [inline src/modules/buttons/Checked.js]: is it really a button?

File: src/modules/buttons/Copy.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```
Comment (nik-kor) [inline src/modules/buttons/Copy.js]: Maybe have a `Control` component?

File: src/modules/Layout/Table/Table.scss [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  
+  &__td {
+    vertical-align: middle;
+    height: 100%;
+
+    &:first-child {
+      font-weight: 700;
+    }
+
+    &-inner {
```
Comment (nik-kor) [inline src/modules/Layout/Table/Table.scss]: can we apply the styles to the cell(td)?

File: src/modules/Control/Control.scss [hunk 5]
```diff
@@ -0,0 +1,25 @@
+.j-control {
+  display: inline-block;
+  width: 20px;
```
Comment (Vxplus) [inline src/modules/Control/Control.scss]: if the image needs to be proportional lets make this a block level variable.

File: src/modules/Control/Control.scss [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  height: 20px;
+  background-repeat: no-repeat;
+
+  &--type {
+
+    &-edit {
```
Comment (Vxplus) [inline src/modules/Control/Control.scss]: also maybe lets have a map with all the types and images path and lets this be generated through that.

File: src/modules/Layout/Table/Table.scss [hunk 7]
```diff
@@ -0,0 +1,57 @@
+@import 'index.scss';
+
+.j-table {
+  width: 100%;
+  text-align: left;
+
+  tbody {
```
Comment (Vxplus) [inline src/modules/Layout/Table/Table.scss]: if we are al ready referencing al the elements with a custom BEM class maybe lets do the same for the body.

File: src/modules/Layout/Table/Table.scss [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+    border: $hf-border;
+  }
+
+  &__row {
+    min-height: 70px;
```
Comment (Vxplus) [inline src/modules/Layout/Table/Table.scss]: seems kind a arbitrary, any specific reason for this?

File: src/modules/Layout/Table/Table.scss [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+    background-color: $hf-color-white;
+
+    &:last-child {
+
+      .j-table__td {
+        padding-bottom: 70px;
```
Comment (Vxplus) [inline src/modules/Layout/Table/Table.scss]: same this amount of padding

File: src/modules/Layout/Table/Table.scss [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+      }
+    }
+  }
+
+  &__th {
+    padding-top: $hf-padding-cell;
+    padding-bottom: $hf-padding-cell;
+    color: $hf-color-gray-medium;
+    font-weight: 700;
```
Comment (Vxplus) [inline src/modules/Layout/Table/Table.scss]: there should be a variable for this and normally the maximum font weight we use is 500.

File: src/modules/Layout/Table/Table.scss [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+      padding-right: $hf-padding-table;
+    }
+  }
+  
+  &__td {
+    vertical-align: middle;
+    height: 100%;
+
+    &:first-child {
+      font-weight: 700;
```
Comment (Vxplus) [inline src/modules/Layout/Table/Table.scss]: same about font weight here.

File: src/modules/Layout/Table/Table.scss (hunk 9 above)
Comment (ivanzusko) [inline src/modules/Layout/Table/Table.scss]: @Vxplus just according to mockup

File: src/modules/Layout/Table/Table.scss (hunk 8 above)
Comment (ivanzusko) [inline src/modules/Layout/Table/Table.scss]: just mockup size. No relations between any other part of the app.

File: src/modules/Layout/Table/Table.scss (hunk 9 above)
Comment (Vxplus) [inline src/modules/Layout/Table/Table.scss]: in that case is the mock covering an edge case or can this side actually be achived by the convo of the padding + content line height?

File: src/modules/pages/ApiListPage/ApiList.jsx [hunk 12]
```diff
@@ -5,7 +5,10 @@ import {
 } from 'react-router-dom';
 import Pagimagic from 'react-pagimagic';
 
+import block from '../../../helpers/bem-cn';
 import Table from '../../Layout/Table/Table';
+import Td from '../../Layout/Table/Td';
```
Comment (nik-kor) [inline src/modules/pages/ApiListPage/ApiList.jsx]: looks like you don't use it. no?

File: src/modules/Control/Control.js [hunk 13]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import block from '../../helpers/bem-cn';
+
+import './Control.css';
+
+const b = block('j-control');
+
+const Control = ({ type }) => {
```
Comment (nik-kor) [inline src/modules/Control/Control.js]: can use `PropTypes.oneOf([])` for `type` property

File: src/modules/Control/Control.js [hunk 14]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```
Comment (nik-kor) [inline src/modules/Control/Control.js]: will it have any callback handlers?

Comment (nik-kor) [inline src/modules/Control/Control.js]: or it's for icons only?

Comment (ivanzusko) [inline src/modules/Control/Control.js]: just for icons only. At least for right now

File: src/modules/Control/Control.js (hunk 13 above)
Comment (ivanzusko) [inline src/modules/Control/Control.js]: Done! Thank you!

File: src/modules/Control/Control.js (hunk 14 above)
Comment (nik-kor) [inline src/modules/Control/Control.js]: then `Icon` is better name

Comment (ivanzusko) [inline src/modules/Control/Control.js]: Yeap, definitely. Done

File: src/modules/Icon/Icon.js [hunk 15]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import block from '../../helpers/bem-cn';
+
+import './Icon.css';
+
+const b = block('j-control');
```
Comment (nik-kor) [inline src/modules/Icon/Icon.js]: should be renamed

Comment (ivanzusko) [inline src/modules/Icon/Icon.js]: @nik-kor done

**Pull request merged**